integration/entity.go

```diff
@@ -23,8 +23,8 @@ type Entity struct {
 
 // EntityMetadata stores entity Metadata
 type EntityMetadata struct {
-	Name string `json:"name"`
-	Type string `json:"type"`
+	Name      string `json:"name"`
+	Namespace string `json:"type"`
```

Comment:
I would add a comment saying that, for compatibility reasons, we keep `type`

Reply:
👍